pokemongo_bot/cell_workers/update_live_stats.py

```diff
@@ -344,6 +343,7 @@ def _get_stats_line(self, player_stats):
             ),
             'pokeballs_thrown': 'Threw {:,} pokeballs'.format(player_stats['pokeballs_thrown']),
             'stardust_earned': 'Earned {:,} Stardust'.format(player_stats['stardust_earned']),
+            'stardust_amount': 'Have {:,} Stardust'.format(player_stats['stardust_amount']),
```

Comment:
'stardust_amount' is not in player_stats. Use self.bot.stardust instead, or metrics.dust.latest.